hw03-testing/src/main/java/com/danzki/hw03/TestingEnvironment/TestingEnv.java

```diff
@@ -0,0 +1,67 @@
+package com.danzki.hw03.TestingEnvironment;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class TestingEnv {
+
+  private static void annotatedMethodLoop(Object obj, Class<?> aClass, Class<?> annotation)
```

Comment:
```suggestion
  private static void executeMethods(Object obj, Class<?> aClass, Class<?> annotation)
```
По названию "цикл по аннотированным методам" - не понятно, что делает этот цикл и какого его назначение

Reply:
Принято